apps/api/src/app/environments/usecases/update-environment/update-environment.usecase.ts

```diff
@@ -20,10 +20,19 @@ export class UpdateEnvironment {
       updatePayload.identifier = command.identifier;
     }
 
-    if (command.dns && command.dns.inboundParseDomain !== '') {
+    if (command.dns && command.dns.inboundParseDomain && command.dns.inboundParseDomain !== '') {
```

Comment:
Added check here to also ensure that the value isn't falsy

Reply:
🚀